doc/src_usermanual/techrep_main.tex

```diff
@@ -784,45 +784,210 @@ \section{Solver options}
 
 \warningcp{Note:} Options set in \texttt{hiop.options} file at runtime overwrite options set at runtime via the above API.
 
+\subsection{Termination criteria and output}
 
+\noindent \textbf{tolerance}: maximum (absolute)  NLP optimality error allowed at the optimal solution. Double values in $[10e-14,0.1]$. Default value: $10e-8$. 

```

Comment:
to keep consistence, should we use $10^{-14}$ and $10^{-8}$ here? See the next option `acceptable_tolerance` for example